maywood/style.css

```diff
@@ -194,8 +194,7 @@ $grid-configuration: map-extend($grid-configuration-default, $grid-configuration
  *   let’s use a placeholder to keep them all
  *   in-sync
  */
-button[data-load-more-btn], button,
-.button,
+button[data-load-more-btn], .button, button,
```

Comment:
These changes also showed up on compile, but weren't modified in the Sass.

Reply:
I think this happens because the internal Sass build has been updated in a recent version and it's now grouping selectors differently. Should be fine.

Reply:
Yeah, this (and the other stuff that pops up after recompiling) is a result of a fix getting added to Varia on `master`, but not yet applied to the child themes. It’s pretty tedious to deploy each theme individually, so sometimes Varia fixes won’t get into a child theme until a theme-specific issue is worked on like in this issue. 

Technically, when we do a fix in Varia we should be auto-recompiling _all_ the child themes on the same branch but that part of the `theme-dev-util` hasn’t got merged to `master` yet so this kind of thing has been happening occasionally. As far as, I’ve seen these kinds of “upstream”, Varia fixes haven’t caused problems, so I think this PR is good to go.